config.temper.md

```diff
@@ -4,9 +4,8 @@ Each example is in its own directory which has the rosettacode.org path so
 rosettacode.org/wiki/Averages/Arithmetic_mean corresponds to the
 sub-directory `Averages/Arithmetic_mean`
 
-    import("./Averages/Arithmetic_mean/example");
-    import("./Binary_search/binary-search");
-    import("./FizzBuzz/fizz-buzz");
-    import("./Greatest_common_divisor/gcd");
-    import("./Matrix_multiplication/dense");
-    import("./Matrix_multiplication/matrix");
+    import("./Averages/Arithmetic_mean");
+    import("./Binary_search");
+    import("./FizzBuzz");
+    import("./Greatest_common_divisor");
+    import("./Matrix_multiplication");
```

Comment:
Only one import for matrix now, so down a line. And the matrix files already had unique names for their types and functions.